src/ceph-detect-init/requirements.txt

```diff
@@ -1 +0,0 @@
-argparse
```

Comment:
Currently `setup.py` adds it to requirements (and installs over network) only if it is not a builtin for the particular Python version.

Also take a look at `setup.py` file in my pull request https://github.com/ceph/ceph/pull/9157